sherpa/optmethods/opt.py

```diff
@@ -20,9 +20,9 @@
 #
 
 import numpy as np
-import multiprocessing
 import random
 from sherpa.utils import Knuth_close, _multi, _ncpus, run_tasks, func_counter
+import multiprocessing
```

Comment:
My pylint keeps on complaining about having Python models after numpy or sherpa modules. This is not a functional change but a style change. If we were to redo it it "should" be

```
import multiprocessing

import numpy as np

from sherpa.utils import kitchen_sink
```

Reply:
(from @taldcroft) - The intent was to be explicit that the first time multiprocessing is imported anywhere is within `sherpa.utils` where the start method is handled. In reality this may not be necessary. IIRC there can be a problem if the start method is changed after actually using multiprocessing, but I suppose that isn't happening here. I'm all for keeping the linters happy when possible, so we can probably revert this.

Reply:
I just reverse this change.